From ef4b1840a05cc22751653b55f0d21dafac7321dc Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Tue, 21 May 2024 07:57:53 -0700
Subject: [PATCH 4/4] hardcoded rocm path

- todo: ROCM_HOME needs to be passed as an environment variable

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 op_builder/builder.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/op_builder/builder.py b/op_builder/builder.py
index 18c13022..8dd45591 100644
--- a/op_builder/builder.py
+++ b/op_builder/builder.py
@@ -235,7 +235,7 @@ class OpBuilder(ABC):
     def get_rocm_gpu_arch():
         if OpBuilder._rocm_gpu_arch:
             return OpBuilder._rocm_gpu_arch
-        rocm_gpu_arch_cmd = "/opt/rocm/bin/rocminfo | grep -o -m 1 'gfx.*'"
+        rocm_gpu_arch_cmd = "/opt/rocm_sdk_611/bin/rocminfo | grep -o -m 1 'gfx.*'"
         try:
             result = subprocess.check_output(rocm_gpu_arch_cmd, shell=True)
             rocm_gpu_arch = result.decode('utf-8').strip()
@@ -248,7 +248,7 @@ class OpBuilder(ABC):
     def get_rocm_wavefront_size():
         if OpBuilder._rocm_wavefront_size:
             return OpBuilder._rocm_wavefront_size
-        rocm_wavefront_size_cmd = "/opt/rocm/bin/rocminfo | grep -Eo -m1 'Wavefront Size:[[:space:]]+[0-9]+' | grep -Eo '[0-9]+'"
+        rocm_wavefront_size_cmd = "/opt/rocm_sdk_611/bin/rocminfo | grep -Eo -m1 'Wavefront Size:[[:space:]]+[0-9]+' | grep -Eo '[0-9]+'"
         try:
             result = subprocess.check_output(rocm_wavefront_size_cmd, shell=True)
             rocm_wavefront_size = result.decode('utf-8').strip()
-- 
2.45.2

